tests/test_orchestrator.py

```diff
@@ -150,6 +165,7 @@ def test_orchestrate_submit_no_validate(self):
 
     def test_orchestrate_with_vcf_files(self):
         with patch('eva_sub_cli.orchestrator.WritableConfig') as m_config, \
+                patch('eva_sub_cli.orchestrator.DEFAULT_METADATA_XLSX_TEMPLATE_VERSION', new='1.1.6'), \
```

Comment:
I'm a bit confused by this, if the metadata template gets updated, presumably it would be updated everywhere including in the tests? In which case we would want the tests to reflect the current state of the code rather than a patched version number.

Reply:
updated to always refer the version from code